usmqe/api/tendrlapi/glusterapi.py

```diff
@@ -71,6 +72,34 @@ def get_volume_list(self, cluster):
         self.check_response(response)
         return response.json()
 
+    def create_bricks(self, cluster, nodes, brick_path, asserts_in=None):
+        """Create volume bricks on given nodes with specified path.
+
+        Name:        "create_bricks",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/GlusterCreateBrick",
+
+        Args:
+            clsuter (str): id of a cluster with nodes
+            nodes (list): ids of nodes where should be bricks
+            brick_path (str): path to brick
+        """
+        pattern = "{}/GlusterCreateBrick".format(cluster)
+        path, brick = brick_path.rsplit(os.sep, 1)
```

Comment:
`os.path.split` is the method you want - see https://docs.python.org/2/library/os.path.html
It doesn't mean that your solution is wrong, it's not.